eth/beacon/types/active_states.py

```diff
@@ -35,8 +35,8 @@ class ActiveState(rlp.Serializable):
     ]
 
     def __init__(self,
-                 pending_attestations: Iterable[AttestationRecord]=None,
-                 recent_block_hashes: Iterable[Hash32]=None) -> None:
+                 pending_attestations: Iterable[AttestationRecord] = None,
+                 recent_block_hashes: Iterable[Hash32] = None) -> None:
```

Comment:
I prefer to not have these white space changes in this PR mainly because of two things:

1. The majority of the code base does **not** use white space which means the change lowers the consistency  overall (just searched for `=None,` vs `= None,`)
2. It shouldn't be the concern of this PR (means, even **if** we would want the white space changes they should be in a separate PR)

Notice that I'm not pro or contra white space. I'm just in favor of consistency.  In fact I hope that in the long run we'll be using something like `black` to enforce consistency.

Reply:
on it, actually autoPEP8 added spaces.